cpp/plugins/cucim.kit.cuslide/src/cuslide/jpeg2k/libopenjpeg.cpp

```diff
@@ -1,18 +1,6 @@
 /*
- * Apache License, Version 2.0
- * Copyright 2021 NVIDIA Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * SPDX-FileCopyrightText: Copyright (c) 2021-2025, NVIDIA CORPORATION
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Please see https://github.com/rapidsai/cucim/pull/960/files#diff-125d2ee27350911a3d015445c4aa3d667224ff04aa50bf76ae3ef68e8db46c80R33-R37 below. Part of this file is derived from the openjpeg's code.

Would the following modification makes sense?

```suggestion
 * SPDX-FileCopyrightText: Copyright (c) 2002-2014, Universite catholique de Louvain (UCL), Belgium
 * SPDX-FileCopyrightText: Copyright (c) 2002-2014, Professor Benoit Macq
 * SPDX-FileCopyrightText: Copyright (c) 2003-2014, Antonin Descampe
 * SPDX-FileCopyrightText: Copyright (c) 2003-2009, Francois-Olivier Devaux
 * SPDX-FileCopyrightText: Copyright (c) 2005, Herve Drolon, FreeImage Team
 * SPDX-FileCopyrightText: Copyright (c) 2002-2003, Yannick Verschueren
 * SPDX-FileCopyrightText: Copyright (c) 2001-2003, David Janssens
 * SPDX-FileCopyrightText: Copyright (c) 2011-2012, Centre National d'Etudes Spatiales (CNES), France
 * SPDX-FileCopyrightText: Copyright (c) 2012, CS Systemes d'Information, France
 * SPDX-FileCopyrightText: Copyright (c) 2021-2025, NVIDIA CORPORATION
 * SPDX-License-Identifier: BSD-2-Clause
 * SPDX-License-Identifier: Apache-2.0
```